spec/spec_helper.rb

```diff
@@ -0,0 +1,100 @@
+# This file was generated by the `rspec --init` command. Conventionally, all
```

Comment:
In the future (*don't bother change it now*) I recommend committing related files together. Eg: Commit this file along with the tests.

That way someone can look at the commit history and see a nice history of what you added step by step.